deploy/crds/kieapp.crd.yaml

```diff
@@ -438,6 +438,13 @@ spec:
                           kieServerJmsAMQQueues:
                             type: string
                             description: AMQ broker broker comma separated queues, if empty the values from default queues will be used.
+                      prometheus:
+                        type: object
+                        description: Configuration for Prometheus Server with KIE Server
+                        properties:
+                          disablePrometheusExt:
+                            type: boolean
+                            description: If set to false, the prometheus server extension will be enabled. (Sets the org.kie.prometheus.server.ext.disabled system property)
```

Comment:
Sorry to be a pain, but if we do want to do this and don't care about having 3 states (undefined), then no need for a prometheus object, unless you expect further configuration in the future that you want to group under it. Otherwise, the disablePrometheusExt directly in there would work.